documentation/schema.md

```diff
@@ -202,6 +203,18 @@ Like the other input-related steps, `select`s can list either one or many hashes
 
 A click_on step terminates the preceding list of input-related steps, by submitting the web form. It is a list containing a hash with only two possible keys: `selector` and `value`. `selector` is the CSS selector for finding the button or link to click, and `value` is the HTML value attribute if present, both to disambiguate and for the benefit of clients which may be POSTing directly instead of using a headless browser, though this is not recommended. `selector` is the only attribute you must provide/should expect to be guaranteed.
 
+### javascript
+
+A javascript step allows running arbitrary client-side javascript on the page that yamltron is on.
```

Comment:
there is also a `values` property, which allows you to inject values into the commands
the `selectors` property is used to inject an array of elements and the `values` property is used to inject an array of values (accessed via `values` within a command). this is useful for injecting our $ variables